bin/vaultclient

```diff
@@ -98,28 +100,13 @@ program
     .command('create-account')
     .option('--name <NAME>')
     .option('--email <EMAIL>')
-    .option('--password <PASSWORD>')
     .action(action.bind(null, 'create-account', (client, args) => {
         client.createAccount(
-            args.name, { email: args.email, password: args.password },
+            args.name, { email: args.email },
             handleVaultResponse);
     }));
 
 program
-    .command('list-account-users')
```

Comment:
[There it is](http://docs.aws.amazon.com/IAM/latest/APIReference/API_ListAccessKeys.html)

I think this should only be done when the equivalent functionality is supported in vault.